Track Your Reps/AppDelegate.swift

```diff
@@ -9,26 +9,31 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     var window: UIWindow?
     
+    var userState: String? {
+        return UserDefaults.standard.string(forKey: "state")
+    }
+    
     static var delegate: AppDelegate? {
         return UIApplication.shared.delegate as? AppDelegate
     }
     
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        let tabBarController = TabBarController() as UIViewController
-        let navigationController = UINavigationController(rootViewController: tabBarController)
-        navigationController.navigationBar.isTranslucent = false
-        navigationController.navigationBar.barTintColor = Palette.pink.color
-        setupWindow(with: navigationController)
+        if userState == nil {
+            let onboardVC = OnBoardViewController()
+            setupWindow(with: onboardVC)
+        } else {
+            let tabBarController = TabBarController()
+            setupWindow(with: tabBarController)
+        }
```

Comment:
could shorten to `initialView = usereState == nil ? OnBoardViewController() : TabBarController(); setupWindow(with: tabBarController)`